### PR TITLE
Final style

### PR DESCRIPTION
**Pull request by anderCM**
- Add any missing JSX elements to your project (without adding extra functionality)
- Your final design should match Zeplin's design (this includes all elements + the same styling)
- You may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/Routes/Books/Book.jsx [hunk 1]
```diff
@@ -10,12 +12,22 @@ const Book = ({ idBook, book }) => {
...
+            </div>
+            <div className="col-4 col-md-3 mt-3">
+              <Progress />
+            </div>
+            <div className="col-8 col-md-3 mt-3">
+              <Chapter />
+            </div>
+          </div>
         </div>
       </div>
```
Comment (DammyShittu) [inline src/Routes/Books/Book.jsx]: - [x] Kindly improve the styling of the book. We are to style the component as exactly as it is in the Zeplin design. Kindly compare the pictures below and try to improve your font styles.

- [x] You are missing the demarcation (vertical straight line) between the left and right sections of the boo card as we have on the design.

- [x] Kindly ensure the font on the progress bar section (text with % sign) should match the design font. The progress bar should also be moved further away from the right section to the left.

- [x] Kindly match the font on the leftmost section to the design, including the text size. The section listing the links should be blue in color as we have on the design.

- [x] The section with the update progress button should have matching font with the design.

- [x] Kindly improve the navbar font as well. It should also be smaller.

  Your Implementation | Zeplin Design
  :---:|:---:
  ruwa | dioq

File: src/Routes/Books/AddBook.jsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ const AddBook = () => {
 
   return (
     <div className="container-fluid mt-5">
-      <h4 className="text-muted">ADD NEW BOOK</h4>
+      <h4 className="text-muted fw-bold">ADD NEW BOOK</h4>
       <form onSubmit={handleSubmit}>
         <div className="row">
           <div className="col-md-4 my-2">
```
Comment (DammyShittu) [inline src/Routes/Books/AddBook.jsx]: - [x] Kindly improve the styling of the form component. The input boxes and button heights should be reduced to match the design

- [x] The button should also be longer with enough space on the left and right of the text.

  Your Implementation | Zeplin Design
  :---:|:---:
  ywa | ioqa

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @anderCM,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Routes/Books/Books.jsx [hunk 3]
```diff
@@ -24,9 +24,9 @@ const Books = () => {
 
   return (
     <>
-      <div className="container border rounded my-3 p-3">
-        { Object.keys(books).map((key) => (
-          <Book key={key} idBook={key} book={books[key][0]} />
+      <div className="container my-3">
+        { Object.keys(books).map((key, index) => (
+          <Book key={key} idBook={key} book={books[key][0]} index={index} />
```
Comment (topeogunleye) [inline src/Routes/Books/Books.jsx:29]: - [x] As required by the previous reviewer, 
 Kindly match the font on the left part of the design. The section listing the links should be blue in color as we have on the design:
 
 
image

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @anderCM, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @anderCM👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Routes/Books/Book.jsx (hunk 1 above)
Comment (anderCM) [reply src/Routes/Books/Book.jsx]: Thanks for the information, I am changing the style to match with template

File: src/Routes/Books/AddBook.jsx (hunk 2 above)
Comment (anderCM) [reply src/Routes/Books/AddBook.jsx]: Thanks for the information, I am changing the style to match with template

File: src/Routes/Books/Books.jsx (hunk 3 above)
Comment (anderCM) [reply src/Routes/Books/Books.jsx:29]: Thanks for the information, I am changing the style to match with template

**Pull request merged**